sdk/src/wallet/error.rs

```diff
@@ -47,7 +47,7 @@ pub enum Error {
     #[error("failed to get remainder address")]
     FailedToGetRemainder,
     /// Insufficient funds to send transaction.
-    #[error("insufficient funds {available}/{required} available")]
+    #[error("Address owns insufficient funds. {required} glow required, but {available} glow available.")]
```

Comment:
Do you really want to start with a capital letter?
Also `glow` is bad here, in IOTA its called `micros`

Reply:
Well not sure if micros is better, since that's wrong in shimmer :trollface:

Reply:
`base unit` might cover everything. What's the problem with capitals btw?

Reply:
We usually don't put capitals because the error is embedded in another and it looks weird

Reply:
We extract it and we always receive the sentences in small caps. This is not an issue, because we need translations for it anyway.

Reply:
It's called base unit in this [blogpost](https://wiki.iota.org/tips/tips/TIP-0032/#iota-stakers---80). 

![image](https://github.com/iotaledger/iota-sdk/assets/45079109/0fcd31a2-857c-4918-a438-132603a7b4d9)